CHANGELOG.md

```diff
@@ -1403,3 +1403,4 @@ Notable changes to the `alacritty_terminal` crate are documented in its
 - Clear the visible region when the RIS escape sequence (`echo -ne '\033c'`) is received
 - Prevent logger from crashing Alacritty when stdout/stderr is not available
 - Fix a crash when sending the IL escape sequence with a large number of lines
+- Fix rendering of rounded corners with builtin box drawing
```

Comment:
Should be moved into the current `-dev` section. Also, just thinking about it, it's likely should be in `### Changed` and use the `Improve rendering...` instead of `Fix rendering...`.